Apps/Admin/Tests/Services/SupportServiceTests.cs

```diff
@@ -98,10 +101,15 @@ public class SupportServiceTests
             bool includeBlockedDataSources,
             string? expectedBlockedDataSources,
             bool includeCovidDetails,
-            bool expectedCovidDetails)
+            bool expectedCovidDetails,
+            ClientRegistryType queryType)
         {
             // Arrange
-            PatientDetailsQuery patientQuery = new() { Hdid = Hdid, Source = PatientDetailSource.All, UseCache = false };
+            PatientDetailsQuery patientQuery = new()
+            {
+                Hdid = queryType == ClientRegistryType.Hdid ? Hdid : null, Phn = queryType == ClientRegistryType.Phn ? Phn : null,
+                Source = queryType == ClientRegistryType.Hdid ? PatientDetailSource.All : PatientDetailSource.Empi, UseCache = false,
```

Comment:
Suggest splitting this so each of the 4 parameters is on a separate line.

Reply:
Fixed